docs/rules.md

```diff
@@ -388,3 +388,4 @@ TODO: A lot of these are generated so this will need expanded with each unique c
 | :white_check_mark: | warning | Web extension | When default_locale is specified a matching messages.json must exist | | | null | NO_MESSAGES_FILE |
 | :white_check_mark: | warning | Web extension | When _locales directory exists, default_locale must exist | | | null | NO_DEFAULT_LOCALE |
 | :white_check_mark: | warning | Web extension | | | | | UNSAFE_VAR_ASSIGNMENT |
+| :white_check_mark: | warning | Web extension | | | | | DANGEROUS_EVAL |
```

Comment:
tbh, that table get's refactored soon anyway when old rules are deleted so I don't want to care too much about this for now…